### ISSUE TITLE
Squash all migrations before production deployment

### ISSUE THREAD

**Issue by fricklerhandwerk** [Nix-Security-WG/nix-security-tracker#376]
We have accumulated ~40 migrations, some of which are fairly expensive. Squash them before deployment, otherwise we will have to run through them on every deployment.

**Comment by Erethon** [comment]
Is this really necessary currently? Even though Django can squash migraton, the [documentation](https://docs.djangoproject.com/en/5.2/topics/migrations/#squashing-migrations) mentions this wouldn't be a problem and if you ever want to do it it's usually when you have multiple hundreds of migrations (which we're not close to having).

Django doesn't re-run migrations on every deployment, it knows which migrations have run and only runs the new ones.

**Comment by fricklerhandwerk** [comment]
> Django doesn't re-run migrations on every deployment, it knows which migrations have run and only runs the new ones.

That's true but it runs them on every test, because we start with a fresh deployment. That could be avoided by providing a database dump, but that opens a different can of worms: how to keep an up-to-date one?

**Comment by fricklerhandwerk** [comment]
Closing as not planned as it's not critical. Migrations take O(seconds) in tests, but since it's not a show stopper (yet), let's think about it again when it hurts more. And before that we should rather set up metrics such as code coverage, CI turnaround time (partitioned by what it's spend on), etc.
